azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/calling/lobby/LobbyOverlayViewModel.kt

```diff
@@ -3,15 +3,20 @@
 
 package com.azure.android.communication.ui.presentation.fragment.calling.lobby
 
+import com.azure.android.communication.ui.configuration.LocalizationProvider
 import com.azure.android.communication.ui.redux.state.CallingStatus
 import kotlinx.coroutines.flow.MutableStateFlow
 import kotlinx.coroutines.flow.StateFlow
 
-internal class LobbyOverlayViewModel {
+internal class LobbyOverlayViewModel(private val appLocalizationProvider: LocalizationProvider) {
     private lateinit var displayLobbyOverlayFlow: MutableStateFlow<Boolean>
 
     fun getDisplayLobbyOverlayFlow(): StateFlow<Boolean> = displayLobbyOverlayFlow
 
+    fun getApplicationLocalizationProvider(): LocalizationProvider {
```

Comment:
getApplicationLocalizationProvider -> getLocalizationProvider

Reply:
Same for the other ViewModel classes, just do not want to add same comment everywhere